IPython/core/interactiveshell.py

```diff
@@ -482,7 +487,9 @@ def __init__(self, ipython_dir=None, profile_dir=None,
 
         # This is where traits with a config_key argument are updated
         # from the values on config.
+        self._preinit = True
```

Comment:
Any reason this is set to `True` then `False`? 

Figured it would be faster to ask than track it down myself.